caddyconfig/httpcaddyfile/directives.go

```diff
@@ -69,6 +69,18 @@ var directiveOrder = []string{
 	"acme_server",
 }
 
+// directiveShouldSortReversed specifies whether a particular
+// directive should be sorted in reverse, compared to the
+// default of most-specific path matcher ordered first.
+// This is typically relevant for directives that have
+// "override" behaviour, where you want the most specific
```

Comment:
Sigh... golangci is trying to make me American.

> [golangci] reported by reviewdog 🐶
> `behaviour` is a misspelling of `behavior` (misspell)

https://dictionary.cambridge.org/us/dictionary/english/behaviour

```suggestion
// "override" behavior, where you want the most specific
```

Reply:
Ha, that's annoying. Sorry!